resources/js/admin.js

```diff
@@ -95,4 +95,17 @@ $(function() {
             }
         });
     });
+    $('option').mousedown(function(e) {
+        e.preventDefault();
+        var originalScrollTop = $(this).parent().scrollTop();

```

Comment:
Please use `const` instead of `var`. You can also use `yarn eslint` to check your js code.

Reply:
Fixed.